includes/restapi/class-notification-controller.php

```diff
@@ -72,7 +82,7 @@ public function authenticate() : bool {
 	 *
 	 * @return WP_REST_RESPONSE|WP_Error REST response or WP Error
 	 */
-	public function get_notifications( $request ) {
+	public function get_items( $request ) {
```

Comment:
Hmm, do you reckon we can add do a union type for the return value on our targeted PHP versions, but maybe

```php
public function get_items( WP_REST_Request $request ) {
```

Reply:
My logic was to follow core, and the comment is how they add types. To me it is odd if we have half the type in the comment and half in the signature. But I can add it in.

Reply:
@Sephsekla that actually breaks the typing in my editor (VSCode), because it doesn't pick up types from a mix of comments and function signature. If I add `WP_REST_Request` it wants the return type to be in the function signature too, if it's added it breaks linting because unions are not supported.